llvm/test/Transforms/SimplifyCFG/switch-to-select-two-case.ll

```diff
@@ -1,5 +1,5 @@
-; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
-; RUN: opt < %s -passes=simplifycfg -simplifycfg-require-and-preserve-domtree=1 -S | FileCheck %s
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --check-globals
+; RUN: opt < %s -passes=prof-inject,simplifycfg -profcheck-weights-for-test -simplifycfg-require-and-preserve-domtree=1 -S | FileCheck %s
```

Comment:
Note: this test is perfect in that it covers all the cases in the change (verified with some appropriately - placed `dbgs()`). To avoid cumbersomely adding `!prof` everywhere, we're using the feature introduced in the previous patch.